src/asynqp/__init__.py

```diff
@@ -19,8 +19,12 @@
 def connect(host='localhost',
             port=5672,
             username='guest', password='guest',
-            virtual_host='/', *,
-            loop=None, sock=None, **kwargs):
+            virtual_host='/',
+            *,
+            loop=None,
+            sock=None,
+            nodelay=True,
```

Comment:
I'm trying to plan ahead. When you're designing public APIs it pays to consider how flexible the solution is.

What happens when the next feature request comes in to set a default for another option? Do we add another keyword parameter? How long can that go on for? Do we change the meaning of the existing parameter and break backwards compatibility?

I'd prefer to have something general. That way, when the next person says "I need to be able to set socket option X", I can just tell them to use the existing API, rather than continue changing my code.